src/coreclr/src/vm/dllimportcallback.cpp

```diff
@@ -1034,8 +1034,16 @@ PCODE TheUMEntryPrestubWorker(UMEntryThunk * pUMEntryThunk)
     if (pThread->IsAbortRequested())
         pThread->HandleThreadAbort();
 
+#if defined(HOST_OSX) && defined(HOST_ARM64)
```

Comment:
This is too broad. The JITed code can be executing inside `UMEntryThunk::DoRunTimeInit`.

Reply:
Fixed